Source/Reporting/Web/src/app/Reporting/CaseReportsForListing/case-report-list.component.scss

```diff
@@ -52,6 +52,19 @@ th {
     }
 }
 
+.table>tbody>tr>td,
+.table>tbody>tr>th {
+  border-top: none;
+}
+tr:hover {
+    background-color: #f9f9f9;
+    color: #403E10;
+   
+    transition-delay: 0s;
```

Comment:
Can be written as shorthand: https://developer.mozilla.org/en-US/docs/Web/CSS/transition